Week1/homework/app.js

```diff
@@ -1,11 +1,156 @@
+    },
+    pride_and_prejudice: {
+      title: 'Pride and Prejudice',
+      language: 'English',
+      author: 'Jane Austen',
+      about:
+        'One of the most famous novels of all time, Pride And Prejudice details the courtship of two opposed characters in a world where manners and courtesy are of the utmost importance.'
+    },
+    the_diary_of_a_young_girl: {
+      title: 'The Diary Of A Young Girl',
+      language: 'English',
+      author: 'Anne Frank',
+      about:
+        'Unforgettable and deeply influential, Anne Frank’s diary is a raw account of a young girl’s life as she hides from the Nazis. Despite her circumstances, Anne believes that people are still good at heart and that the world is full of beauty: she will change your life.'
+    },
+    the_book_thief: {
+      title: 'The Book Thief',
+      language: 'English',
+      author: 'Markus Zusak',
+      about:
+        'Set in Germany during 1939, The Book Thief follows Liesel as she rescues books from the tyranny of Nazi rule. Meanwhile, her family has hidden a Jewish fighter in their basement and death looks down on the family, narrating our tale. Experience bravery that is rarely found in the world, and friendship that is formed in the most unlikely of situations.'
+    },
+    the_hobbit: {
+      title: 'The Hobbit',
+      language: 'English',
+      author: 'J.R.R. Tolkien',
+      about:
+        'Although the movies are inexplicably long, The Hobbit was originally written as a short children’s book. Meet your favourite characters for the first time as the unforgettable Bilbo Baggins traverses the harsh landscapes of Middle Earth to challenge a dragon.'
+    },
+    little_women: {
+      title: 'Little Women',
+      language: 'English',
+      author: 'Louisa May Alcott',
+      about:
+        'Join four sisters, each with their own prominent personality, as they come of age in charming 19th Century New England. Experience their struggles and revel in their flaws, as these girls become strong women.'
+    }
+  };
+
+  // Step 1.7
+  const bookCoversLocation = {
+    to_kill_a_mockingbird: './img/to_kill_a_mockingbird.jpg',
+    1984: './img/1984.jpg',
+    harry_potter_and_the_philosophers_stone: './img/harry_potter_and_the_philosophers_stone.jpg',
+    the_lord_of_the_rings: './img/the_lord_of_the_rings.jpg',
+    the_great_gatsby: './img/the_great_gatsby.jpg',
+    pride_and_prejudice: './img/pride_and_prejudice.png',
+    the_diary_of_a_young_girl: './img/the_diary_of_a_young_girl.jpg',
+    the_book_thief: './img/the_book_thief.jpg',
+    the_hobbit: './img/the_hobbit.jpg',
+    little_women: './img/little_women.jpg'
+  };
+
+  // Step 1.3 - 1.5
+  function htmlForBooks() {
+    const ol = document.createElement('ol');
+    for (let i = 0; i < bookTitles.length; i++) {
```

Comment:
Since you are using the index variable `i` only in line 119 to get the current book title, you can also use a `for...of` loop and get rid of line 119:

```js
for (const bookTitle of bookTitles) {
```